MAUI/Busy-Indicator/Set-Header.md

```diff
@@ -0,0 +1,321 @@
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                SfBusyIndicator busyIndicator = new SfBusyIndicator()
+                {
+                    IsRunning = true,
+                    AnimationType = AnimationType.CircularMaterial,
+                    Title = "Loading...",
+                    TextColor = Colors.Red,
+                    TitlePlacement = BusyIndicatorTitlePlacement.Top
+                };
+                this.Content = busyIndicator;
+            }
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+The following screenshot illustrates the result of the above code.
+
+![Title Placement](Images/SetHeader/TitlePlacement.png)
+
+
+## TitleSpacing
+
+SfBusyIndicator provides options to set the space between the indicator and the title. The space can be set using the `TitleSpacing` property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:core="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
+                x:Class="BusyIndicatorSample.MainPage">
+
+        <ContentPage.Content>
+            <core:SfBusyIndicator x:Name="busyindicator"
+                                  IsRunning="True"
+                                  AnimationType="CircularMaterial"
+                                  Title="Loading..."
+                                  TextColor = "Red" 
+                                  TitlePlacement="Top"
+                                  TitleSpacing="20"/>           
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.Core;
+    using Microsoft.Maui;
+    using Microsoft.Maui.Controls;
+
+    namespace BusyIndicatorSample
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                SfBusyIndicator busyIndicator = new SfBusyIndicator()
+                {
+                    IsRunning = true,
+                    AnimationType = AnimationType.CircularMaterial,
+                    Title = "Loading...",
+                    TextColor = Colors.Red,
+                    TitlePlacement = BusyIndicatorTitlePlacement.Top,
+                    TitleSpacing = 20
+                };
+                this.Content = busyIndicator;
+            }
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+The following screenshot illustrates the result of the above code.
+
+![TitleSpacing](Images/SetHeader/Spacing.png)
+
+
+
+## Font customization
+
+SfBusyIndicator provides options to customize the title text.  The title text can be customized using the `FontSize`, `FontAttributes` and `FontFamily` properties.
```

Comment:
SfBusyIndicator provides options to customize the title text. The title text can be customized using the `FontSize,` `FontAttributes,` and `FontFamily` properties.